src/service/plugins/sms.js

```diff
@@ -85,7 +85,7 @@ var Metadata = {
  *
  * TEXT_MESSAGE: Has a "body" field which contains pure, human-readable text
  */
-var MessageEvent = {
+var MessageEventType = {
```

Comment:
Seems unrelated?

Reply:
The reason for this is that `MessageEvent` is defined in the `shared-node-browser` env

It seems gjs doesn't provide `MessageEvent`, but it's just easier and imo preferable to rename this variable instead of conflicting with a standardised API name